build.xml

```diff
@@ -154,15 +155,13 @@
         </exec>
     </target>
 
-    <target name="phpstan" depends="tests-acceptance-build" description="Performs static analysis of PHP files using PHPStan on all packages.">
+    <target name="phpstan-packages" depends="tests-acceptance-build" description="Performs static analysis of PHP files using PHPStan on all packages.">
```

Comment:
```suggestion
    <target name="phpstan-packages" depends="phpstan" description="Performs static analysis of PHP files using PHPStan on all packages.">
```

Reply:
we could spare duplicate execution of `tests-acceptance-build` target since project-base is also package.